app/components/task/holder.hbs

```diff
@@ -7,11 +7,12 @@
           id="input"
           type="range"
           name="percentCompleted"
-          value="{{@task.percentCompleted}}"
+          disabled={{@disabled}}
+          value="{{this.percentCompleted}}"
```

Comment:
I don't quotes are required here. Let me know your view on this

Reply:
yes, they are not required, thanks